File: SRC - PERL/00 - Estruturas Sequenciais/11.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba dois números maiores que zero, calcule e mostre um elevado ao outro

print "Digite um número maior que 0 > ";
my $numero1 = <>;
chomp $numero1;

print "Digite outro número maior que 0 > ";
my $numero2 = <>;
chomp $numero2;

my $pot1 = $numero1 ** $numero2;
my $pot2 = $numero2 ** $numero1;

printf "O número %d elevado a %d é %d \n", $numero1, $numero2, $pot1;
printf "O número %d elevado a %d é %d \n", $numero2, $numero1, $pot2;

File: startingPoint/02-EstruturaSequencial/Exercicios_Propostos/11.py
#!/usr/bin/python
# coding: latin-1

# 11 - Faça um programa que receba dois números maiores que zero, calcule e mostre um elevado ao outro

num1 = int(input("Digite o primeiro número positivo: "))
num2 = int(input("Digite o segundo número positivo: "))

r = num1 ** num2
r1 = num2 ** num1

print ("O primeiro número elevado ao segundo número é: ", r, "\nO segundo número elevado ao primeiro número é: ", r1)
